en/debug/mavlink_shell.md

```diff
@@ -0,0 +1,39 @@
+# MAVLink Shell
+
+The MAVLink Shell is an *NSH console* that can be accessed via MAVLink over serial (USB/Telemetry) or WiFi (UDP/TCP) links (in particular, on NuttX-based systems like: Pixhawk, Pixracer, etc.).
+
+The shell can be used for running commands and modules (and displaying the output of modules that it starts). 
+It cannot display the output of tasks running on the work queue (or modules that it does not start).
+
+> **Tip** The [QGroundControl MAVLink Console](#qgroundcontrol) is the easiest way to access the console.
+  If the system does not start properly you should instead use the [System Console](../debug/system_console.md).
+
+## Opening the Shell
+
+### QGroundControl MAVLink Console {#qgroundcontrol}
+
+The easiest way to access shell is to use the [QGroundControl MAVLink Console](https://docs.qgroundcontrol.com/en/analyze_view/mavlink_console.html) (see **Analyze View > Mavlink Console**).
+
```

Comment:
Probably will extend this with an image. Looks a bit silly as just a section with a sentence.

Reply:
But not now!